about.html

```diff
@@ -0,0 +1,154 @@
+            <ion-icon name="close-outline" class="mobile-nav-icon "></ion-icon>
+        </div>
+        <div class="left-nav">
+            <div class="logo">
+                <a href="index.html"><img src="image/UI-LOGO-BLUE-final.png" alt=""></a>
+            </div>
+
+            <div class="Home"><a href="index.html">Home</a></div>
+            <div class="about"><a href="about.html">About</a></div>
+            <div class="contactUs"><a href="contactus.html">Contact Us</a></div>
+            <div class="docs"><a href="documentation.html">Documentation</a></div>
+        </div>
+        <div class="right-nav">
+            <div class="linkedin"><a href="https://linkedin.com/in/saurabhmaurya45"><img src="image/icons8-linkedin-48.png" alt="linkedin"></a></div>
+            <div class="git-logo"><a href="https://github.com/saurabhmaurya45/ui-component-library"><img src="image/git-logo-f.png" alt="Github"></a></div>
+            <div class="dark-mode"><img src="image/dark-mode-logo.png" alt="dark-mode"></div>
+        </div>
+
+    </div>
+
+
+    <!-- navbar end -->
+
+    <!-- section-1 code start -->
+    <section class="about">
+        <div class="container">
+            <h2>About Our Library</h2>
+            <p>Welcome to our UI Component Library, where innovation meets simplicity. In a world filled with various UI libraries, we identified a need for a comprehensive solution that aligns with modern development practices. This library is our answer, crafted to enhance your development experience.</p>
+
+            <h3>Why Opt for Our Component Library?</h3>
+            <ul>
+                <li>We believe your time should be invested in creating extraordinary projects that solve real-world problems and benefit others.</li>
+                <li>No more repetitive tasks – our library eliminates the need to recreate components repeatedly, letting you focus on what matters most.</li>
+                <li>Embrace the DRY (Don't Repeat Yourself) principles not just in code but in your development workflow as a whole! 😄</li>
+            </ul>
+
+            <p>While we acknowledge that no single library can cater to all projects, we're committed to expanding and evolving. Your suggestions are invaluable; feel free to contribute by forking the repository, creating pull requests, or opening issues. Thanks to everyone who has contributed to making this library even better!</p>
+        </div>
+    </section>
+
+    <section class="built-with">
+        <div class="container">
+            <h2>Technologies We Used</h2>
+            <ul>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+            </ul>
+        </div>
+    </section>
+
+
+    <!-- section-1 code end -->
+
+
+    <footer>
+      <div class="f1">
+          <div class="footer-icons">
+              <div class="footer-icon facebook">
+                  <a href=""><i class="fab fa-facebook-f"></i></a>
```

Comment:
give urls as well

Reply:
sir, it has been updated. Url is present there